### ISSUE TITLE
Error on eslint-config-airbnb v13.0.0

### ISSUE THREAD

**Issue by wellyshen** [airbnb/javascript#1172]
Hi Guys, I have updated my eslint-config-airbnb to v13.0.0, when I run `npm test` the following error occurs, I have installed "eslint-plugin-jsx-a11y", does anyone meet this problem?

> Module build failed: Error: /Users/Welly/Desktop/react-cool-starter/node_modules/eslint-config-airbnb/rules/react-a11y.js:
	Configuration for rule "jsx-a11y/anchor-has-content" is invalid:
	Value "" is the wrong type.

**Comment by ljharb** [comment]
If you use the install command in the readme, you'd end up with the proper version of all the peer deps- including v2 (not v3) of eslint-plugin-jsx-a11y.

**Comment by msikma** [comment]
There might be a problem in the ESLint website [getting started guide](http://eslint.org/docs/user-guide/getting-started). (I presume that's the page you mean by readme, but correct me if I'm wrong.) Or in the installer script, possibly.

I probably have to send this to the ESLint page though, so my apologies if this is in the wrong place.

I've just set up ESLint for a new project which didn't have any kind of linting before, and I ended up with v3 as well. Basically, I installed `eslint` (which installed v3.9.1), ran `./node_modules/.bin/eslint --init`, then chose Airbnb and JSON.

This yields an install of `eslint-plugin-jsx-a11y@3.0.1` (specifically that version in the ), and this complaint:

    npm WARN eslint-config-airbnb@13.0.0 requires a peer of eslint-plugin-jsx-a11y@^2.2.3 but none was installed.

Linting any file produces the same error as reported by @wellyshen, and manually removing jsx-a11y and reinstalling `^2.2.3` fixes the problem.

**Comment by gergoerdosi** [comment]
Not that one, @ljharb refers to the command here:

https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb#eslint-config-airbnb-1

**Comment by msikma** [comment]
In that case I'll take this to ESLint since their out of the box configuration for Airbnb must be broken.

**Comment by wellyshen** [comment]
@ljharb Fixed by the command, thanks a lot :)

**Comment by ljharb** [comment]
Indeed, `eslint --init` will cause this bug - see eslint issue https://github.com/eslint/eslint/issues/7338.

**Comment by karolis-inntec** [comment]
Also this issue looks relevant https://github.com/eslint/eslint/issues/8547
